readme.md

```diff
@@ -0,0 +1,24 @@
+# wp-qingstor
```

Comment:
Filename should be `README.md`

Reply:
Modified already.

Reply:
`README.md` in uppercase instead of `readme.md`